verification/curator-service/ui/src/components/ViewCase.tsx

```diff
@@ -107,10 +108,20 @@ const useStyles = makeStyles((theme) => ({
     container: {
         marginTop: '1em',
         marginBottom: '1em',
+        marginLeft: '10em',
+        // width - left shift and nav menu size.
+        width: 'calc(100vw - 23em)',
+        scrollBehavior: 'smooth',
```

Comment:
Is this still needed now that we're using the library?

Reply:
yes, default is not smooth.